### PR TITLE
Fixed the Icon overlapping issue

### PR DESCRIPTION
**Pull request by tanujvyas10**
Fixes issue number #357

**Result**
Now the setting and logout icons are aligned

**Screenshot**
settinglog

### CODE REVIEW

**Comment by tanujvyas10** [comment]
@jaskirat2000 @devesh-verma @vaibhavdaren Please review my work.
thank you sir @vaibhavdaren for the help

File: src/user/dashboard/settings/layouts/SettingSidebar.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            </NavLink>
+          </ListGroup.Item>
+          <ListGroup.Item className={location ? "active" : "inactive"}>
+            <NavLink to="/location" className="link">
+              <b><MdLocationOn size={iconsSize}/>Location</b>
+            </NavLink>
+          </ListGroup.Item>
+          <ListGroup.Item
+            className={notification ? "active" : "inactive"}
+          >
```
Comment (vaibhavdaren) [inline src/user/dashboard/settings/layouts/SettingSidebar.js]: for loop

Comment (Rupeshiya) [reply src/user/dashboard/settings/layouts/SettingSidebar.js]: @vaibhavdaren I have already done this part in material-ui branch, and also this PR is on to master branch, so what to do?

**Comment by tanujvyas10** [comment]
@vaibhavdaren created a fresh PR for this issue 
closing this one

**Pull request closed without merging**